pkg/fixtures/fixtures.go

```diff
@@ -105,10 +106,11 @@ func NewFixture(fs afero.Fs, apiKey, stripeAccount, baseURL, file string) (*Fixt
 // defined to populate the user's account
 func (fxt *Fixture) Execute() error {
 	for _, data := range fxt.fixture.Fixtures {
+		fmt.Println()
```

Comment:
is that needed?

Reply:
It was added to make it a bit easier to differentiate the outputs for the events that run multiple fixtures. However, with a simpler output this will not be needed anymore!